.travis.yml

```diff
@@ -11,7 +11,7 @@ install:
   - wget -O $HOME/purescript.tar.gz https://github.com/purescript/purescript/releases/download/$TAG/linux64.tar.gz
   - tar -xvf $HOME/purescript.tar.gz -C $HOME/
   - chmod a+x $HOME/purescript
-  - npm install -g bower pulp purescript-psa
+  - npm install -g bower pulp purescript-psa psc-package
```

Comment:
Nice, good simple interim solution to fix CI until I stop being lazy and set up a nixjs binary cache :+1: